aces/Loris
# [Tools] Add script to delete and refresh raisinbread data

## Brief summary of changes

This script will dump your raisinbread database and then re-import the data. It automates the steps in [raisinbread/README.md](https://github.com/aces/Loris/tree/minor/raisinbread).

* It will also restore your `url`, `base`, `host` settings so that LORIS won't break if you're not hosting it on `localhost`.

* It will ask you to change the front-end `admin` password so you can login to LORIS without using the public Raisinbread password.

* I've added `make testdata` as a quick way to refresh your database.

* This can also be used upon switching between branches so that you never get an issue where your database is broken because of sync issues.

~~As a precaution I decided to force the user of this script to manually type in the database name of the database they want to effectively delete. This should help prevent data loss. Feel free to suggest a different security mechanism.~~

UPDATE I incorporated Rida's suggestion to force the script to exit if the `sandbox` flag is not set in config.xml.


#### Testing instructions (if applicable)

1. Checkout the branch.
2. run `make testdata` OR go to `tools/` and run `php raisinbread_refresh.php`
3. Verify that you have fresh Raisinbread data and that your front-end still works.

#### Links to related tickets (GitHub, Redmine, ...)

* Resolves #5255.
